ciphers/rsa_factorization.py

```diff
@@ -30,25 +28,33 @@ def rsafactor(d: int, e: int, N: int) -> list[int]:
     [113, 241]
     >>> rsafactor(4021, 13, 17711)
     [89, 199]
+    >>> rsafactor(int("123c5b61ba36edb1d3679904199a89ea80c09b9122e1400c09adcf77846" +
+    ...    "76d01d23356a7d44d6bd8bd50e94bfc723fa87d8862b75177691c11d757692df8881", 16),
+    ...    0x10001,
+    ...    int("a66791dc6988168de7ab77419bb7fb0c001c62710270075142942e19a8d8c51d05" +
+    ...    "3b3e3782a1de5dc5af4ebe99468170114a1dfe67cdc9a9af55d655620bbab", 16))[0]
+    23234950162188993388155927630085331316851060055334470382368804331834850828939
+    >>> rsafactor(4, 3, 13)

```

Comment:
```suggestion
    >>> rsa_factor(4021, 13, 17711)
    [89, 199]
    >>> rsa_factor(int("123c5b61ba36edb1d3679904199a89ea80c09b9122e1400c09adcf77846" +
    ...    "76d01d23356a7d44d6bd8bd50e94bfc723fa87d8862b75177691c11d757692df8881", 16),
    ...    0x10001,
    ...    int("a66791dc6988168de7ab77419bb7fb0c001c62710270075142942e19a8d8c51d05" +
    ...    "3b3e3782a1de5dc5af4ebe99468170114a1dfe67cdc9a9af55d655620bbab", 16))[0]
    23234950162188993388155927630085331316851060055334470382368804331834850828939
    >>> rsa_factor(4, 3, 13)
```
Changing function name to snake case